Create what-is-the-difference-between-unlimited-sites.md (#211)
beaverbuilder/documentation: docs/general/pre-sales/what-is-the-difference-between-unlimited-sites.md

---
id: what-is-the-difference-between-unlimited-sites
title: Difference between Unlimited Sites and Multi-Site capable
sidebar_label: Difference between Unlimited Sites and Multi-Site capable
---

> # Difference between Unlimited Sites and Multi Site capable

Reviewer: Use intro text instead of heading:
```suggestion
Here's the difference between *multisite-capable* and *unlimited sites* in relation to Beaver Builder packages and your Beavver Builder license.
```

## Multi Site Capable

> Normally for any site you want to use WordPress on, you install it separately so it's a standalone installation. Multi Site capable refers to a specific WordPress setup that lets you install WordPress once and manage multiple sites under that one installation (also referred to as a network).  You can read more about Network/Multisite setups over in the [WordPress documentation](https://wordpress.org/support/article/create-a-network/).  Only our Pro and Agency packages are Multi Site Capable.

Reviewer: ```suggestion
Normally, you install WordPress on each site you build so it's a standalone installation. *Multisite-capable* refers to a specific WordPress setup that lets you install WordPress once and manage multiple sites under that one installation. This is also referred to as a network.  You can read more about network/multisite setups in the [WordPress documentation](https://wordpress.org/support/article/create-a-network/).  

Only the Beaver Builder Pro and Agency packages are multisite-capable.
```

> ## Unlimited Sites

Reviewer: ```suggestion
## Unlimited sites
```

> Unlimited Sites refers to the number of sites that you can add your license key to.  There is no limit on any of our packages and that includes for Multi Site setups.  You also can use your license key across servers and even hosts.  

Reviewer: ```suggestion
*Unlimited sites* refers to the number of sites that you can add your license key to.  There is no limit on any of our packages, including network sites with multisite setups.  

*Unlimited sites* also means that you can use your license key across servers and even hosts.  

See our [Terms and Conditions](https://www.wpbeaverbuilder.com/terms-and-conditions/) for details about the conditions under which you can use your license key.
```
